build.gradle.kts

```diff
@@ -13,7 +13,7 @@ plugins {
 }
 
 scmVersion {
-    versionCreator("versionWithBranch")
+    unshallowRepoOnCI.set(true)
```

Comment:
i think we can set it as default now, it's been working fine ;) !

Reply:
First I would like to fix multiple "Unshallowing repo because build is being executed on CI" logs, I have that on my radar 😉